autoware_internal_planning_msgs/msg/ScoredCandidateTrajectory.msg

```diff
@@ -0,0 +1,4 @@
+std_msgs/Header header
+unique_identifier_msgs/UUID generator_id
+autoware_planning_msgs/TrajectoryPoint[] points
```

Comment:
```suggestion
autoware_internal_planning_msgs/CandidateTrajectory candidate_trajectory
```

To avoid duplication, it could be reused.

Reply:
If the header is needed for synchronization message filters, a stamped version of this message could be created.

Reply:
Good suggestion.